src/main/resources/openapi/cases.yaml

```diff
@@ -625,7 +625,7 @@ components:
           example: "Sentencing remarks"
         requested_on:
           type: string
-          format: date
+          format: date-time
           example: 2022-05-22
```

Comment:
Looks like the 2 examples need to change to be date-time also?

Reply:
hearings and cases yaml files updated to reflect 'requested_on' examples as 2023-06-20T10:00:00Z